components/public-api/gitpod/v1/scm.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service SCMService {
+  rpc GetToken(GetTokenRequest) returns (GetTokenResponse) {}
+  rpc GuessTokenScopes(GuessTokenScopesRequest) returns (GuessTokenScopesResponse) {}
+  rpc SearchRepositories(SearchRepositoriesRequest) returns (SearchRepositoriesResponse) {}
+  rpc ListSuggestedRepositories(ListSuggestedRepositoriesRequest) returns (ListSuggestedRepositoriesResponse) {}
```

Comment:
Would be really a good opportunity to combine `serachRepositories` and. `listSuggestedRepositories` in one method.

Reply:
Let me see if we can make it happen without significant changes to dashboard. That said, I'd prefer the proven rollout strategy with the compat ship on FE to any API overhaul that would disallow us from rolling out with a safety net.

Reply:
Thanks but I think we should avoid that for now, it has a too many implications.